README.md

```diff
@@ -1,6 +1,7 @@
 # Build environment for Pelion Edge firmware
 
-This repo contains a set of instructions (this README) and some scripts for building the Pelion OS Edge firmware image.
+This repo contains a set of instructions (this README) and some scripts for building the Pelion Edge firmware image.
+When used in conjuction with the [Repo Manifest](https://github.com/armpelionedge/manifest-pelion-edge) repository, it automates the complete [Yocto build instructions](https://github.com/armpelionedge/meta-pelion-edge/blob/dev/BUILD.md).
```

Comment:
this points to dev branch on the manifest repo.  should this point to master instead?